src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+            }
+            System.out.println("\t" + HOR_LINE + System.lineSeparator());
+    }
+
+    /**
+     * Prints acknowledgement message that item has been added to list
+     * and also mentions the total no. of tasks in list currently.
+     *
+     * @param currTask Task object that was just created after user input.
+     */
+    private static void printAddedResponse(Task currTask) {
+        System.out.println("\t" + HOR_LINE);
+        System.out.println("\tGot it. I've added this task: ");
+        System.out.println("\t" + currTask.getTypeIcon() + currTask.getStatusIcon() + currTask.description);
+        System.out.println("\tNow there are " + (additions + 1) + " tasks in the list.");
+        System.out.println("\t" + HOR_LINE + System.lineSeparator());
+    }
+
+    /**
+     * Depending on the starting keyword present in user input, creates respective
+     * Task subclass object and stores it inside the storedTasks array.
+     *
+     * @param userInput String command input by user.
+     */
+    public static void createTask(String userInput) {
+        if (userInput.startsWith("event ")) {
+            storedTasks[additions] = new Event(userInput);
+        }
+        else if (userInput.startsWith("deadline")) {
+            storedTasks[additions] = new Deadline(userInput);
+        }
+        else if (userInput.startsWith("todo ")) {
+            storedTasks[additions] = new Todo(userInput);
+        }
+
+        printAddedResponse(storedTasks[additions]);
+        additions++;
+    }
+
+    /**
+     * Executes Task mode, where user's commands are added into to-do list.
+     * When "list" entered, prints out current to-do list.
+     * When "completed <task id>" or "done <task id>" entered, updates that task status.
+     * When "remove <task id>" or "clear <task id>" entered, removes that task from list.
+     * When "change" is typed in by user, switches program to Echo mode.
+     */
+    public static void startTask() {
+        Scanner in = new Scanner(System.in);
+        String toAdd = in.nextLine();
+        String textLowerC = toAdd.toLowerCase();
+
+        while (!textLowerC.equals("bye") & !textLowerC.equals("list") & !textLowerC.equals("change")) {
+            defaultTaskModeBehavior(toAdd, textLowerC);
+            toAdd = in.nextLine();
+            textLowerC = toAdd.toLowerCase();
+        }
+
+        if (textLowerC.equals("list")) {
+            printList();
+            startTask();
+        }
+
+        // Switch to Echo mode if user types in "change"
+        if (textLowerC.equals("change")) {
+            switchMode(ECHO_MODE);
+        }
+    }
+
+    /**
+     * Checks different scenarios for input commands of user in Task mode and
+     * does the necessary action.
+     *
+     * @param toAdd String input by user.
+     * @param textLowerC String input by user converted to lower cases for checking.
+     */
+    private static void defaultTaskModeBehavior(String toAdd, String textLowerC) {
```

Comment:
Perhaps rename this method to start with a verb? As per the coding standard, "names representing methods must be verbs and written in camelCase." (https://se-education.org/guides/conventions/java/basic.html#naming)

Reply:
Ok will rectify in next commit.